src/microbe_stage/Microbe.Interior.cs

```diff
@@ -1190,6 +1190,9 @@ private void HandleOsmoregulation(float delta)
             osmoregulationCost *= 20.0f / (20.0f + Colony.ColonyMembers.Count);
         }
 
+        if (CellTypeProperties.IsBacteria == false)
```

Comment:
Explicitly checking bools with a bool value is not good style (should be just `if (CellTypeProperties.IsBacteria)`)

Reply:
So if I need the boolean to be false, what should I do instead?

An else?

Reply:
`if (!CellTypeProperties.IsBacteria)`